app/src/main/java/com/konkuk/codion/ui/common/filter/FilterBottomSheetButton.kt

```diff
@@ -0,0 +1,69 @@
+package com.konkuk.codion.ui.common.filter
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.konkuk.codion.R
+import com.konkuk.codion.ui.theme.CodiOnTypography
+import com.konkuk.codion.ui.theme.Gray500
+
+@Composable
+fun FilterBottomSheetButton(
+    placeholder: String,
+    onClick: () -> Unit
+) {
+    Box(
+        modifier = Modifier
+            .border(1.dp, Gray500, RoundedCornerShape(2.dp))
```

Comment:
피그마에는 weight 0.5로 stroke가 설정되어 있습니닷
```suggestion
            .border(0.5.dp, Gray500, RoundedCornerShape(2.dp))
```

Reply:
피그마처럼 0.5로 하면 너무 안보이길래 1로 설정했습니두 ..!

Reply:
프리뷰로 봐서 1dp가 두꺼워 보였나봐욥
1dp 좋습니닷